common/presenters/move-to-card-component.js

```diff
@@ -5,16 +5,18 @@ const personToCardComponent = require('./person-to-card-component')
 function moveToCardComponent({ showMeta = true, showTags = true } = {}) {
   return function item({ id, reference, person = {}, status }) {
     const href = `/move/${id}`
+    const excludedBadgeStatuses = ['cancelled']
+    const statusBadge = excludedBadgeStatuses.includes(status)
```

Comment:
Maybe `const statusBadge = excludedBadgeStatuses.includes(status) &&  { text: i18n.t(`statuses::${status}`) }` is slightly more expressive? I tend to use it instead of ternaries with an undefined. But it's a matter of personal taste, and many people don't like it.

Reply:
Ah, forget the above, it becomes `false`, of course, not `undefined`.